distribution/Directory.Build.props

```diff
@@ -35,12 +40,12 @@
   <!-- Shared code analyzers used for all projects in the solution -->
   <ItemGroup Label="Code Analyzers">
     <PackageReference Include="AsyncFixer" Version="1.5.1" PrivateAssets="All" />
-    <PackageReference Include="Asyncify" Version="0.9.7" PrivateAssets="All" />
-    <PackageReference Include="Meziantou.Analyzer" Version="1.0.606" PrivateAssets="All" />
+    <PackageReference Include="Asyncify" Version="0.9.7" PrivateAssets="all" />
+    <PackageReference Include="Meziantou.Analyzer" Version="1.0.606" PrivateAssets="all" />
```

Comment:
Is there a reason for lower "all" in some special cases?

Reply:
No - it's just not relevant as long as it is spelled correctly.